src/Test.UnitTests.Sarif.Driver/Test.UnitTests.Sarif.Driver.csproj

```diff
@@ -8,7 +8,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks>net48;net6.0</TargetFrameworks>
```

Comment:
>net48 [](http://example.com/codeflow?start=22&length=5)

A key change. We need to run more tests under .NET 4.8 because this framework is a primary production use case for us.